### DISCUSSION TITLE
Neovim Copilot suggestion appends binary characters once selected

### DISCUSSION THREAD

**Discussion by Marko-Sanchez** [community/community#60372] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Neovim: v0.9.1
Copilot.nvim: v1.9.1
OS: Ubuntu 22.02.2 LTS x86_64
Gnome Terminal

While changing the default mappings for copilot, changed tab to control-right, I encountered a bug where auto completion would append characters `<80>@7 ` to the end of the accepted response. Issue is seen with `<C-Right>` and `<A-a>, <C-a>`; Although, when using the default `<Tab>` the bug does not appear.
![image](https://github.com/community/community/assets/32421124/eaad1216-7598-4c67-83f1-97f92f9fc6ba)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Marko-Sanchez ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by sontixyou** [comment, marked as answer]
Maybe add this code, problem is resolve.


add  `replace_keycodes = false` in copilot#accept keymap config

**Reply by Marko-Sanchez** [reply]
thank you!
